src/main/java/org/ow2/petals/deploymentmodel/serviceunitmodel/ServiceUnit.java

```diff
@@ -0,0 +1,67 @@
+package org.ow2.petals.deploymentmodel.serviceunitmodel;
+
+import java.net.URI;
+import java.util.Map;
+
+import org.ow2.petals.deploymentmodel.busmodel.ComponentInstance;
+
+/**
+ * Class describing a service unit to deploy.
+ *
+ * @author alagane
+ */
+public class ServiceUnit {
+    /**
+     * The unique identifier of this service unit in the model. Can be different from the service unit name.
+     */
+    private String id;
+
+    /**
+     * The URL of the associated archive that can be a service assembly or a deployable service unit.
+     */
+    private URI url;
+
+    /**
+     * The map of placeholder required by the service unit. The key of the map is the key of the placeholder, and the
+     * value of the map is the default value of the placeholder. If there is no default value, the value must be null.
+     */
+    private Map<String, String> placeholders;
```

Comment:
Plutôt une list de `Placeholder`, non ? Car sinon où est stockée la description du placeholder ? Pour info, la description peut être utilisée en mode interractif pour saisir la valeur du `PlaceholderInstance`

Reply:
To simplify the model, I decided (after discussing with @vincent-zurczak) to remove the description. It could be added in the future if needed.

Reply:
ok